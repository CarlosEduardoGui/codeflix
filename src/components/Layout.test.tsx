import { render } from "@testing-library/react"
import { Layout } from "./Layout"
import { BrowserRouter } from "react-router-dom"

describe("Layout", () => {
    it("should render layout correctly", () => {
        const { asFragment } = render(
        <Layout>
            <div>Test</div>
        </Layout>, {
            wrapper: BrowserRouter
        })

        expect(asFragment()).toMatchSnapshot();
    })
})