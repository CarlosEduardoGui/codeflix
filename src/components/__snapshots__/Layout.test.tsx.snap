// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout should render layout correctly 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-0"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg css-1km29jt-MuiContainer-root"
    >
      <div>
        Test
      </div>
    </div>
  </div>
</DocumentFragment>
`;
